perf/perf_uprobe.py

```diff
@@ -44,8 +44,12 @@ def setUp(self):
         if 'Ubuntu' in self.distro_name:
             deps.extend(['linux-tools-common', 'linux-tools-%s' %
                          platform.uname()[2]])
-        elif self.distro_name in ['rhel', 'SuSE', 'fedora', 'centos']:
+        elif self.distro_name in ['SuSE', 'fedora', 'centos']:
             deps.extend(['perf'])
+        elif self.distro_name == 'rhel' and int(self.detected_distro.version) < 9:
+            deps.extend(['perf'])
```

Comment:
still not looks good , may https://github.com/avocado-framework-tests/avocado-misc-tests/pull/2135/files#diff-fda9f5351c1380644151553ec90f0a432a82cac6faeae5046e16589826474cbaR47 this need  revert to the old way and